### PR TITLE
[Get-DbaDbPartitionScheme] - adds filtering by name

### PR DESCRIPTION
**Pull request by MikeyBronowski**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [x] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [x] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [x] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/dataplat/appveyor-lab ?
 - [ ] Unit test is included
 - [ ] Documentation
 - [ ] Build system

### Commands to test

`Get-DbaDbPartitionScheme -SqlInstance localhost -Database TestDB -PartitionScheme partSch01`

### CODE REVIEW

**Comment by wsmelton** [comment]
You body of the pull request didn't explain why the changes are being made. If you do not have an issue to link those details, please include them.

**Comment by potatoqualitee** [comment]
This looks reasonable to me, thank you 👍🏼  I will merge after forcing some test re-runs

**Review by potatoqualitee — APPROVED** [review]

**Comment by potatoqualitee** [comment]
ahhh unfortunately, the test appears to be legititmately failing 

https://ci.appveyor.com/project/dataplat/dbatools/builds/42643971/job/d7ld3gtyo5evr52o

```
Describe : Get-DbaDbPartitionScheme Unit Tests
Context  : Validate parameters
Name     : It Should only contain our specific parameters
Result   : Failed
Message  : Expected 0, but got 1.
Describe : Remove-DbaDbPartitionScheme Unit Tests
Context  : Validate parameters
Name     : It Should only contain our specific parameters
Result   : Failed
Message  : Expected $null or empty, but got @(@{InputObject=PartitionScheme; SideIndicator=<=}).

```

File: functions/Get-DbaDbPartitionScheme.ps1 [hunk 1]
```diff
@@ -66,6 +74,8 @@ function Get-DbaDbPartitionScheme {
         [PSCredential]$SqlCredential,
         [object[]]$Database,
         [object[]]$ExcludeDatabase,
+        [Alias("Name")]
+        [string[]]$PartitionScheme,
```
Comment (potatoqualitee) [inline functions/Get-DbaDbPartitionScheme.ps1:78]: partition scheme is added to Get but only Remove is tested

**Review by potatoqualitee — CHANGES_REQUESTED** [review]
tests or code need to be updated

File: functions/Get-DbaDbPartitionScheme.ps1 (hunk 1 above)
Comment (MikeyBronowski) [reply functions/Get-DbaDbPartitionScheme.ps1:78]: That was my attempt to reset all the things :) I think it's sorted now. Sorry for the delay.

**Comment by potatoqualitee** [comment]
thank you 😊

File: functions/Get-DbaDbPartitionScheme.ps1 (hunk 1 above)
Comment (potatoqualitee) [reply functions/Get-DbaDbPartitionScheme.ps1:78]: no problem for the delay, thanks for the enhancement🫂

**Pull request merged**